tests/test_models.py

```diff
@@ -260,13 +260,18 @@ def test_rgc_save_load(self, tmp_path):
 
     def test_frontend(self):
         im = po.make_basic_stimuli()
-        frontend = po.simul.Front_End()
+        frontend = po.simul.FrontEnd(pretrained=True, requires_grad=False)
         frontend(im)
 
+    def test_frontend_display_filters(self):
+        frontend = po.simul.FrontEnd(pretrained=True, requires_grad=False)
+        frontend.display_filters()
+        plt.close('all')
```

Comment:
Same as below comment: can we test that this also works with `pretrained=False` and `requires_grad=True` (separately and together).

Reply:
done.